packages/paste-website/src/components/site-wrapper/site-header/SiteHeaderLogo.tsx

```diff
@@ -1,22 +1,34 @@
 import * as React from 'react';
-import {Link} from 'gatsby';
 import {trackCustomEvent} from 'gatsby-plugin-google-analytics';
 import {Text} from '@twilio-paste/text';
 import {Box} from '@twilio-paste/box';
 import {MediaObject, MediaBody, MediaFigure} from '@twilio-paste/media-object';
-import {styled} from '@twilio-paste/styling-library';
-// import {useTheme} from '@twilio-paste/theme';
-// import {PasteIconInverse} from '../../icons/PasteIconInverse';
+import {useTheme} from '@twilio-paste/theme';
+import {PasteIcon} from '../../icons/PasteIcon';
 import {PasteIconPride} from '../../icons/PasteIconPride';
 
-const StyledLink = styled(Link)`
-  text-decoration: none;
-  color: inherit;
+interface StyledLinkProps {
+  href: string;
+  onClick?: () => void;
+  onMouseEnter?: () => void;
+  onMouseLeave?: () => void;
+}
 
-  &:visited {
-    color: inherit;
-  }
-`;
+const StyledLink: React.FC<StyledLinkProps> = ({children, ...props}) => {
+  return (
+    <Box
+      {...props}
+      as="a"
```

Comment:
If we use the gatsby link we can benefit from JS routing here which won't cause a full page refresh. Not sure if this is a big deal, so consider it a nit being called out.

Reply:
I was wondering about this, some places on the site we use Gatsby Link and sometimes we just use anchor. Do we want to have a stance on this as we're refactoring so many parts of the site?

Reply:
lol, we do kind of have a stance, we just made some mistakes and missed somethings.

If we're internal linking, we should be using the `SiteLink` component which is a wrapper around Gatsby Link, or if we don't want it to be blue, a custom link using gatsby link.

Reply:
Updated to use Gatsby's link component, but opted to not use `SiteLink` as the UI effects didn't match between how we treat normal text links and how we wanted to treat this header link.